src/lib.rs

```diff
@@ -0,0 +1,3 @@
+pub mod resample_tools;
```

Comment:
what this files does?

Reply:
It tells the rust compiler to create a module called "resample_tools" from `src/resample_tools.rs` available from this library crate

Reply:
Which is used for example in this doctest
https://github.com/demoth/cpucat/pull/3/files#diff-385473d0d79ed1ce67e90fea85b9f990680beefd4aa9bd3a2ba38b81e6bff1abR29-R37

Reply:
Ah you mean the `lib.rs`? I tried to explain in the commit message. https://github.com/demoth/cpucat/pull/3/commits/8727c205e53265a3631cf1383eef568dba86f104